doc/index.rst

```diff
@@ -0,0 +1,25 @@
+.. gem-plotting-tools documentation master file, created by
```

Comment:
`gem-plotting-tools` should bechanged to `gemplotting`, reflecting the actual name of the `python` package